src/components/ReportActionItem/IOUQuote.js

```diff
@@ -76,14 +75,14 @@ const IOUQuote = props => (
                 <Text style={[styles.flex1, styles.mr2]}>
                     <Text style={props.shouldAllowViewDetails && styles.chatItemMessageLink}>
                         {/* Get first word of IOU message */}
-                        {Str.htmlDecode(fragment.text.split(' ')[0])}
+                        {fragment.text.split(' ')[0]}
                     </Text>
                     <Text style={[styles.chatItemMessage, props.shouldAllowViewDetails
                         ? styles.cursorPointer
                         : styles.cursorDefault]}
                     >
                         {/* Get remainder of IOU message */}
-                        {Str.htmlDecode(fragment.text.substring(fragment.text.indexOf(' ')))}
+                        {fragment.text.substring(fragment.text.indexOf(' '))}
```

Comment:
This causes a regression on staging https://staging.new.expensify.com when cancelling a money request with IOU message contains special html characters. See  https://expensify.slack.com/archives/C049HHMV9SM/p1680254182283689 

@pecanoro @sobitneupane @tienifr Should we fix this before it's deployed to production?

Reply:
Yeah, we should

Reply:
Oh it's already in production

Reply:
@eh2077 I was unable to reproduce the issue on staging v1.2.93-2. Can you try reproducing it again?

Reply:
Did you try 1.2.93-4?

Reply:
Oh yes. I was able to reproduce. @tienifr let's get this sorted asap.